### ISSUE TITLE
lsregistration does not attempt IPv4 on init attempts

### ISSUE THREAD

**Issue by igarny** [perfsonar/ls-registration-daemon#48]
on CentOS 6:

lsregistrationdaemon.log:
```
2018/04/17 05:55:03 (25536) INFO> lsregistrationdaemon.pl:257 main:: - Initial LS URL set to
2018/04/17 05:55:03 (25536) ERROR> lsregistrationdaemon.pl:301 main:: - Unable to determine ls_instance so not performing any operations
2018/04/17 05:55:03 (25536) DEBUG> lsregistrationdaemon.pl:307 main:: - Time until next record refresh is 3600 seconds
2018/04/17 06:55:03 (25536) DEBUG> lsregistrationdaemon.pl:200 main:: - No latency threshold specified for switching LSes. Defaulting to 10%
2018/04/17 06:55:03 (25536) DEBUG> Utils.pm:119 perfSONAR_PS::Client::Utils::send_http_request - Sending HTTP GET to http://ps1.es.net:8096/lookup/activehosts.json


2018/04/17 06:55:08 (25536) ERROR> LookupService.pm:110 perfSONAR_PS::Utils::LookupService::discover_lookup_services - Problem retrieving http://ps1.es.net:8096/lookup/activehosts.json: 500 timeout


2018/04/17 06:55:08 (25536) WARN> lsregistrationdaemon.pl:144 main::__ANON__ - Warned: Use of uninitialized value $current_ls_instance in concatenation (.) or string at /usr/lib/perfsonar/bin/lsregistrationdaemon.pl line 257.

```

the tests:
```
test-rhps02 ~]$ curl -6 http://ps1.es.net:8096/lookup/activehosts.json
curl: (7) couldn't connect to host

test-rhps02 ~]$ curl -4 http://ps1.es.net:8096/lookup/activehosts.json
{"hosts":[{"priority":100,"locator":"http://ps-west.es.net:8090/lookup/records","status":"alive"},{"priority":100,"locator":"http://ps-east.es.net:8090/lookup/records","status":"alive"},{"priority":100,"locator":"http://monipe-ls.rnp.br:8090/lookup/records","status":"alive"},{"priority":100,"locator":"http://ps-sls.sanren.ac.za:8090/lookup/records","status":"alive"},{"priority":100,"locator":"http://nsw-brwy-sls1.aarnet.net.au:8090/lookup/records/","status":"alive"}]}
```

**Comment by arlake228** [comment]
I _think_  this is fixed in 4.1. LWP is notoriously bad at IPv6. In 4.1 we replaced LWP with a different library correctly falls back and has fixed most of our IPv6 issues. I've mostly looked at it from a pSConfig perspective, but the ls-registrationdaemon uses the same library so it should see the same improvements. We should verify this.

**Comment by arlake228** [comment]
I think this is fixed with the lib change, we can re-open if not.
